blog/02-09-2022-configuring-a-logger-for-the-go-sdk.mdx

```diff
@@ -0,0 +1,107 @@
+---
+slug: configuring-a-logger-for-the-go-sdk
+title: Configuring a logger for the golang-sdk
+authors: [skyerus]
+tags: [golang, blog, tutorial, golang-sdk, hooks, logging, logger]
+unlisted: false
```

Comment:
```suggestion
draft: false
```

`unlisted` doesn't seem to do anything... `draft: true` hides the blog in the production build. Just updating this to prevent confusion.